### PR TITLE
Basic bootstrap BE with AWS express and postgres

### PR DESCRIPTION
**Pull request by hromogen**
Add basic set up with deployable express pg lambda
Add documentation

### CODE REVIEW

File: packages/server/REDME.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+## Previous steps
```
Comment (karolinayanko) [inline packages/server/REDME.md]: typo in file name README.md please fix

File: packages/server/package.json [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
-}
\ No newline at end of file
+    "core": "0.0.1",
+    "aws-serverless-express": "^3.3.8",
+    "express": "^4.17.1",
+    "pg": "^8.2.1"
+  },
+  "scripts": {
+    "start:psql:local": "psql -U postgres -h 127.0.0.1 -p qwerty1234 -d postgres ",
+    "sart:dev": "yarn start:psql:local && yarn tsc && yarn nodemon local",
```
Comment (dartwlad) [inline packages/server/package.json]: start

File: packages/server/REDME.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- install `NodeJS` to your local machine
+
+- install `postgresql` to your local machine
+
+- install dependencies in the folder
+
+## Development process
+
+#### sart:dev
```
Comment (karolinayanko) [inline packages/server/REDME.md]: start:dev

File: packages/server/package.json (hunk 2 above)
Comment (karolinayanko) [inline packages/server/package.json]: start:dev

File: packages/server/REDME.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+- starts a postgres DB locally via sart:dev
```
Comment (karolinayanko) [inline packages/server/REDME.md]: start:dev

**Review by dubbha — APPROVED** [review]

**Review by dubbha — APPROVED** [review]

**Review by dartwlad — APPROVED** [review]
In general, it's ok, but in future when we would start working with encryption we would face issue with bcrypt lib again.

**Review by karolinayanko — APPROVED** [review]

**Pull request merged**